Backend/api/models.py

```diff
@@ -9,3 +9,10 @@ class Alumni(User):
     
     def __str__(self):
         return self.name
+
+
+class ElectiveCourse(models.Model):
+    name = models.CharField(max_length=100)
```

Comment:
first no new line at the end in many files
second what happens if the max_length is exceeded ? can you put in comments what does it do